balena/models/organization.py

```diff
@@ -161,3 +164,241 @@ def remove(self, org_id):
+            endpoint=self.settings.get('pine_endpoint'), login=True
+        )['d']
+
+        if len(org_membership) > 0:
+            return org_membership[0]
+        else:
+            raise exceptions.OrganizationMembershipNotFound(membership_id)
+
+
+class OrganizationMembershipTag(BaseTag):
+    """
+    This class implements organization membership tag model for balena python SDK.
+
+    """
+
+    def __init__(self):
+        super(OrganizationMembershipTag, self).__init__('organization_membership')
+
+    def get_all_by_organization(self, handle_or_id):
+        """
+        Get all organization membership tags for an organization.
+
+        Args:
+            handle_or_id (str): organization handle (string) or id (number).
+
+        Returns:
+            list: organization membership tags.
+
+        Raises:
+            OrganizationNotFound: if organization couldn't be found.
+
+        Examples:
+            >>> balena.models.organization.membership.tag.get_all_by_organization(3014)
+            [{'id': 991, 'organization_membership': {'__id': 17608, '__deferred': {'uri': '/resin/organization_membership(@id)?@id=17608'}}, 'tag_key': 'mTag1', 'value': 'Python SDK 1', '__metadata': {'uri': '/resin/organization_membership_tag(@id)?@id=991'}}]
+
+        """
+
+        org_id = handle_or_id
+
+        if not is_id(handle_or_id):
+            params1 = {
+                'filter': 'handle',
+                'eq': handle_or_id
+            }
+
+            org = self.base_request.request(
+                'organization', 'GET', params=params1,
+                endpoint=self.settings.get('pine_endpoint'), login=True
+            )['d']
+
+            if not org:
+                raise exceptions.OrganizationNotFound(handle_or_id)
+
+            org_id = org['id']
+
+        query = '$filter=organization_membership/any(d:d/is_member_of__organization%20eq%20{org_id})'.format(org_id=org_id)
+
+        return super(OrganizationMembershipTag, self).get_all(raw_query=query)
+
+    def get_all_by_organization_membership(self, membership_id):
+        """
+        Get all organization membership tags for all memberships of an organization.
+
+        Args:
+            membership_id (str): organization membership id.
+
+        Returns:
+            list: organization membership tags.
+
+        Examples:
+            >>> balena.models.organization.membership.tag.get_all_by_organization_membership(17608)
+            [{'id': 991, 'organization_membership': {'__id': 17608, '__deferred': {'uri': '/resin/organization_membership(@id)?@id=17608'}}, 'tag_key': 'mTag1', 'value': 'Python SDK 1', '__metadata': {'uri': '/resin/organization_membership_tag(@id)?@id=991'}}]
+
+        """
+
+        params = {
+            'filter': 'organization_membership',
+            'eq': membership_id
+        }
+
+        return super(OrganizationMembershipTag, self).get_all(params=params)
+
+    def get_all(self):
```

Comment:
How about also accepting a `params` argument here?
* In general we do allow that in all getAll method in the JS sdk
* If we do so, then all other methods could use the getAll of the current instance, instead of the base class.